ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -6,6 +6,10 @@
 - hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
+    - name: Call predeploy
```

Comment:
True, I changed this the other way around to acomplish @raukadah suggestions, but, if I end implementing the skip deploying edpm computes in case ```cifmw_libvirt_manager_configuration.vms.compute.amount``` this change has no sense.